### PR TITLE
add item into cart and fix ajax commit

### PR DESCRIPTION
**Pull request by Doantrungdung2001**
[Task #68755](https://edu-redmine.sun-asterisk.vn/issues/68755)

### CODE REVIEW

**Comment by Doantrungdung2001** [comment]
ready

File: src/main/java/com/example/naitei19javaecommerce/controller/user/CartController.java [hunk 1]
```diff
@@ -30,30 +41,63 @@ public String index(Model model) {
...
     }
 
+
+    //add item into cart
+    @PostMapping(value = "/add")
+    public String create(Model model,
+                         @RequestParam("id") Long id,
+                         @RequestParam("quantity") int quantity) {
+        Long userId = userService.getUserisLogin().getId();
+        System.out.println(userId);
```
Comment (vanvtt-0952) [inline src/main/java/com/example/naitei19javaecommerce/controller/user/CartController.java]:
```suggestion
```

File: src/main/java/com/example/naitei19javaecommerce/controller/user/CartController.java [hunk 2]
```diff
@@ -30,30 +41,63 @@ public String index(Model model) {
         return "user/cart/layout-cart";
     }
 
+
+    //add item into cart
+    @PostMapping(value = "/add")
```
Comment (vanvtt-0952) [inline src/main/java/com/example/naitei19javaecommerce/controller/user/CartController.java]:
```suggestion
    @PostMapping(value = "/")
```

**Comment by Doantrungdung2001** [comment]
ready

**Pull request merged**